app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -224,7 +230,13 @@
 - unless %w(Conference Subscription Registration User Organization).include?(version.item_type)
   - if (version.item_type == 'Role' && role_name == 'organization_admin') || (version.item_type == 'UsersRole' && role.name == 'organization_admin')
     in organization
-    = link_to_organization(version.conference_id)
+    - if Organization.find_by(id: version.conference_id)
+      -# organization_admin belongs to organization and not conferences
+      - organization = Organization.find_by(id: version.conference_id)
+      = link_if_alive version, role.name,
```

Comment:
You've changed the content of the link from the organization, to the role_name. This is causing the persistent test failure:

```
expected to find text 
    "added role organization_admin with ID 2 to user name75 in organization Friesen LLC"
in
    "... added role organization_admin with ID 2 to user name75 in organization organization_admin ... "

Reply:
@bear454 Yes you are right! Correcting it! Thanks!

Reply:
@bear454 Please have a look!